cmd/optimizer-nri-plugin/main.go

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (c) 2023. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+	"sigs.k8s.io/yaml"
+
+	"github.com/containerd/nri/pkg/api"
+	"github.com/containerd/nri/pkg/stub"
+	"github.com/containerd/nydus-snapshotter/pkg/errdefs"
+	"github.com/containerd/nydus-snapshotter/pkg/fanotify"
+	"github.com/containerd/nydus-snapshotter/version"
+	"github.com/pelletier/go-toml"
+)
+
+const (
+	defaultLogFile    = "/opt/nri/optimizer/optimizer.log"
```

Comment:
Logs should be stored in `/var/log` or just stream it to syslog service rather than set a file as default on at `/opt`

Reply:
Now, we redirect it to the syslog service.